src/main/java/org/scoula/backend/member/domain/Company.java

```diff
@@ -63,4 +67,21 @@ public class Company {
 
     @Column(name = "list_shrs", nullable = false)
     private String listShrs; // 상장주식수 (LIST_SHRS)
+
+    private BigDecimal closingPrice; // 종가
```

Comment:
```suggestion
    private BigDecimal closingPrice; // 전일종가
```
종가로만 주석 남겨놓으면 헷갈릴것같아서 전일종가로 명시하는건 어떨까요?